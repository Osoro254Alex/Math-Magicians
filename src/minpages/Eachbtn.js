/* eslint-disable */
import React from 'react';

const Eachbtn = (props) => (
  <button type="button" className={props.class} onClick={props.click}>
    {props.text}
  </button>
);

export default Eachbtn;