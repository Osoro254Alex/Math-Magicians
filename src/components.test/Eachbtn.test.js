/* eslint-disable */
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Eachbtn from '../components/Eachbtn';

afterEach(cleanup);

describe('test for button component', () => {
  test('button should render without error', () => {
    render(<Eachbtn />);
  });
  test('matches snapshot for new button', () => {
    const tree = renderer.create(<Eachbtn />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});