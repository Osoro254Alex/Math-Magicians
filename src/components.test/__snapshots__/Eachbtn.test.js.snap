// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for button component matches snapshot for new button 1`] = `
<button
  type="button"
/>
`;
